src/NzbDrone.Core/Notifications/Ntfy/NtfySettings.cs

```diff
@@ -43,16 +43,19 @@ public NtfySettings()
         [FieldDefinition(2, Label = "Password", Type = FieldType.Password, HelpText = "Optional Password", Privacy = PrivacyLevel.Password)]
         public string Password { get; set; }
 
-        [FieldDefinition(3, Label = "Priority", Type = FieldType.Select, SelectOptions = typeof(NtfyPriority))]
+        [FieldDefinition(3, Label = "Access Token", HelpText = "Optional token-based authorization; takes priority over username/password", HelpLink = "https://docs.ntfy.sh/config/#access-tokens")]
```

Comment:
In order to protect the token this should be `Type = FieldType.Password` and `Privacy = PrivacyLevel.ApiKey`